test/src/sdk-harness/build.sh

```diff
@@ -8,7 +8,7 @@ for f in $FILES
 do
 if [ ! -e ${f}/$MANIFEST ]
   then
-  echo "Can't build a project without a Forc manifest."
+  echo ""${f}" not built : can't build a project without a Forc manifest."
```

Comment:
This will conflict with the work in #1372
Maybe we can land that one first and then update this.

Reply:
Oh I forgot I left this in there. Will revert it

Reply:
Resolved by [39e4fba](https://github.com/FuelLabs/sway/pull/1374/commits/39e4fbaaa3481b8acbd05db4744cbb2d4b5f2073)